### ISSUE TITLE
Training accuracy stays at 0

### ISSUE THREAD

**Issue by vinhtq115** [PaddlePaddle/PaddleOCR#1547]
Hi. I'm trying to train an English recognition model (with additional characters) using ICDAR 2015 dataset but the accuracy remains at 0 after 72 epochs. How can I fix this?

Training files in my repo at https://github.com/vinhtq115/PaddleOCR

Some output log:
```
-----------  Configuration Arguments -----------
gpus: 0
heter_worker_num: None
heter_workers: 
http_port: None
ips: 127.0.0.1
log_dir: log
nproc_per_node: None
server_num: None
servers: 
training_script: tools/train.py
training_script_args: ['-c', 'configs/rec/rec_icdar15_train.yml']
worker_num: None
workers: 
------------------------------------------------
WARNING 2020-12-21 16:49:00,035 launch.py:314] Not found distinct arguments and compiled with cuda. Default use collective mode
launch train in GPU mode
INFO 2020-12-21 16:49:00,036 launch_utils.py:473] Local start 1 processes. First process distributed environment info (Only For Debug): 
    +=======================================================================================+
    |                        Distributed Envs                      Value                    |
    +---------------------------------------------------------------------------------------+
    |                       PADDLE_TRAINER_ID                        0                      |
    |                 PADDLE_CURRENT_ENDPOINT                 127.0.0.1:49519               |
    |                     PADDLE_TRAINERS_NUM                        1                      |
    |                PADDLE_TRAINER_ENDPOINTS                 127.0.0.1:49519               |
    |                     FLAGS_selected_gpus                        0                      |
    +=======================================================================================+

INFO 2020-12-21 16:49:00,036 launch_utils.py:476] details abouts PADDLE_TRAINER_ENDPOINTS can be found in log/endpoints.log.
[2020/12/21 16:49:02] root INFO: Architecture : 
[2020/12/21 16:49:02] root INFO:     Backbone : 
[2020/12/21 16:49:02] root INFO:         model_name : large
[2020/12/21 16:49:02] root INFO:         name : MobileNetV3
[2020/12/21 16:49:02] root INFO:         scale : 0.5
[2020/12/21 16:49:02] root INFO:     Head : 
[2020/12/21 16:49:02] root INFO:         fc_decay : 0
[2020/12/21 16:49:02] root INFO:         name : CTCHead
[2020/12/21 16:49:02] root INFO:     Neck : 
[2020/12/21 16:49:02] root INFO:         encoder_type : rnn
[2020/12/21 16:49:02] root INFO:         hidden_size : 96
[2020/12/21 16:49:02] root INFO:         name : SequenceEncoder
[2020/12/21 16:49:02] root INFO:     Transform : None
[2020/12/21 16:49:02] root INFO:     algorithm : CRNN
[2020/12/21 16:49:02] root INFO:     model_type : rec
[2020/12/21 16:49:02] root INFO: Eval : 
[2020/12/21 16:49:02] root INFO:     dataset : 
[2020/12/21 16:49:02] root INFO:         data_dir : ./train_data/
[2020/12/21 16:49:02] root INFO:         label_file_list : ['./train_data/rec_gt_train.txt']
[2020/12/21 16:49:02] root INFO:         name : SimpleDataSet
[2020/12/21 16:49:02] root INFO:         transforms : 
[2020/12/21 16:49:02] root INFO:             DecodeImage : 
[2020/12/21 16:49:02] root INFO:                 channel_first : False
[2020/12/21 16:49:02] root INFO:                 img_mode : BGR
[2020/12/21 16:49:02] root INFO:             CTCLabelEncode : None
[2020/12/21 16:49:02] root INFO:             RecResizeImg : 
[2020/12/21 16:49:02] root INFO:                 image_shape : [3, 32, 100]
[2020/12/21 16:49:02] root INFO:             KeepKeys : 
[2020/12/21 16:49:02] root INFO:                 keep_keys : ['image', 'label', 'length']
[2020/12/21 16:49:02] root INFO:     loader : 
[2020/12/21 16:49:02] root INFO:         batch_size_per_card : 256
[2020/12/21 16:49:02] root INFO:         drop_last : False
[2020/12/21 16:49:02] root INFO:         num_workers : 4
[2020/12/21 16:49:02] root INFO:         shuffle : False
[2020/12/21 16:49:02] root INFO: Global : 
[2020/12/21 16:49:02] root INFO:     cal_metric_during_train : True
[2020/12/21 16:49:02] root INFO:     character_dict_path : ppocr/utils/dict/en_dict_2.txt
[2020/12/21 16:49:02] root INFO:     character_type : ch
[2020/12/21 16:49:02] root INFO:     checkpoints : None
[2020/12/21 16:49:02] root INFO:     debug : False
[2020/12/21 16:49:02] root INFO:     distributed : False
[2020/12/21 16:49:02] root INFO:     epoch_num : 72
[2020/12/21 16:49:02] root INFO:     eval_batch_step : [0, 2000]
[2020/12/21 16:49:02] root INFO:     infer_img : doc/imgs_words_en/word_10.png
[2020/12/21 16:49:02] root INFO:     infer_mode : False
[2020/12/21 16:49:02] root INFO:     log_smooth_window : 20
[2020/12/21 16:49:02] root INFO:     max_text_length : 25
[2020/12/21 16:49:02] root INFO:     pretrained_model : None
[2020/12/21 16:49:02] root INFO:     print_batch_step : 10
[2020/12/21 16:49:02] root INFO:     save_epoch_step : 3
[2020/12/21 16:49:02] root INFO:     save_inference_dir : None
[2020/12/21 16:49:02] root INFO:     save_model_dir : ./output/rec/ic15/
[2020/12/21 16:49:02] root INFO:     use_gpu : True
[2020/12/21 16:49:02] root INFO:     use_space_char : True
[2020/12/21 16:49:02] root INFO:     use_visualdl : False
[2020/12/21 16:49:02] root INFO: Loss : 
[2020/12/21 16:49:02] root INFO:     name : CTCLoss
[2020/12/21 16:49:02] root INFO: Metric : 
[2020/12/21 16:49:02] root INFO:     main_indicator : acc
[2020/12/21 16:49:02] root INFO:     name : RecMetric
[2020/12/21 16:49:02] root INFO: Optimizer : 
[2020/12/21 16:49:02] root INFO:     beta1 : 0.9
[2020/12/21 16:49:02] root INFO:     beta2 : 0.999
[2020/12/21 16:49:02] root INFO:     lr : 
[2020/12/21 16:49:02] root INFO:         learning_rate : 0.0005
[2020/12/21 16:49:02] root INFO:     name : Adam
[2020/12/21 16:49:02] root INFO:     regularizer : 
[2020/12/21 16:49:02] root INFO:         factor : 0
[2020/12/21 16:49:02] root INFO:         name : L2
[2020/12/21 16:49:02] root INFO: PostProcess : 
[2020/12/21 16:49:02] root INFO:     name : CTCLabelDecode
[2020/12/21 16:49:02] root INFO: Train : 
[2020/12/21 16:49:02] root INFO:     dataset : 
[2020/12/21 16:49:02] root INFO:         data_dir : ./train_data/
[2020/12/21 16:49:02] root INFO:         label_file_list : ['./train_data/rec_gt_train.txt']
[2020/12/21 16:49:02] root INFO:         name : SimpleDataSet
[2020/12/21 16:49:02] root INFO:         transforms : 
[2020/12/21 16:49:02] root INFO:             DecodeImage : 
[2020/12/21 16:49:02] root INFO:                 channel_first : False
[2020/12/21 16:49:02] root INFO:                 img_mode : BGR
[2020/12/21 16:49:02] root INFO:             CTCLabelEncode : None
[2020/12/21 16:49:02] root INFO:             RecResizeImg : 
[2020/12/21 16:49:02] root INFO:                 image_shape : [3, 32, 100]
[2020/12/21 16:49:02] root INFO:             KeepKeys : 
[2020/12/21 16:49:02] root INFO:                 keep_keys : ['image', 'label', 'length']
[2020/12/21 16:49:02] root INFO:     loader : 
[2020/12/21 16:49:02] root INFO:         batch_size_per_card : 256
[2020/12/21 16:49:02] root INFO:         drop_last : True
[2020/12/21 16:49:02] root INFO:         num_workers : 8
[2020/12/21 16:49:02] root INFO:         shuffle : True
[2020/12/21 16:49:02] root INFO: train with paddle 2.0.0-rc1 and device CUDAPlace(0)
[2020/12/21 16:49:02] root INFO: Initialize indexs of datasets:['./train_data/rec_gt_train.txt']
[2020/12/21 16:49:02] root INFO: Initialize indexs of datasets:['./train_data/rec_gt_train.txt']
W1221 16:49:02.501819   297 device_context.cc:320] Please NOTE: device: 0, GPU Compute Capability: 7.0, Driver API Version: 10.1, Runtime API Version: 10.1
W1221 16:49:02.958211   297 device_context.cc:330] device: 0, cuDNN Version: 7.6.
[2020/12/21 16:49:10] root INFO: train from scratch
[2020/12/21 16:49:10] root INFO: train dataloader has 17 iters, valid dataloader has 18 iters
[2020/12/21 16:49:10] root INFO: During the training process, after the 0th iteration, an evaluation is run every 2000 iterations
[2020/12/21 16:49:14] root INFO: epoch: [0/72], iter: 10, lr: 0.000500, loss: 75.542274, acc: 0.000000, norm_edit_dis: 0.000000, reader_cost: 0.18198 s, batch_cost: 0.37572 s, samples: 2816, ips: 749.49350
[2020/12/21 16:49:15] root INFO: save model in ./output/rec/ic15/latest
[2020/12/21 16:49:15] root INFO: Initialize indexs of datasets:['./train_data/rec_gt_train.txt']
[2020/12/21 16:49:17] root INFO: epoch: [1/72], iter: 20, lr: 0.000500, loss: 42.426758, acc: 0.000000, norm_edit_dis: 0.000000, reader_cost: 0.08927 s, batch_cost: 0.17549 s, samples: 1024, ips: 583.49707
[2020/12/21 16:49:18] root INFO: epoch: [1/72], iter: 30, lr: 0.000500, loss: 24.794193, acc: 0.000000, norm_edit_dis: 0.000000, reader_cost: 0.00062 s, batch_cost: 0.09069 s, samples: 2560, ips: 2822.87145
[2020/12/21 16:49:19] root INFO: save model in ./output/rec/ic15/latest
[2020/12/21 16:49:19] root INFO: Initialize indexs of datasets:['./train_data/rec_gt_train.txt']
[2020/12/21 16:49:22] root INFO: epoch: [2/72], iter: 40, lr: 0.000500, loss: 23.701382, acc: 0.000000, norm_edit_dis: 0.000000, reader_cost: 0.08479 s, batch_cost: 0.21060 s, samples: 1792, ips: 850.91897
[2020/12/21 16:49:23] root INFO: epoch: [2/72], iter: 50, lr: 0.000500, loss: 22.784916, acc: 0.000000, norm_edit_dis: 0.000000, reader_cost: 0.00018 s, batch_cost: 0.08898 s, samples: 2560, ips: 2876.96985
[2020/12/21 16:49:23] root INFO: save model in ./output/rec/ic15/latest
...
[2020/12/21 16:53:56] root INFO: Initialize indexs of datasets:['./train_data/rec_gt_train.txt']
[2020/12/21 16:53:58] root INFO: epoch: [70/72], iter: 1190, lr: 0.000500, loss: 15.809980, acc: 0.000000, norm_edit_dis: 0.180813, reader_cost: 0.06225 s, batch_cost: 0.12050 s, samples: 256, ips: 212.45016
[2020/12/21 16:53:59] root INFO: epoch: [70/72], iter: 1200, lr: 0.000500, loss: 15.861901, acc: 0.000000, norm_edit_dis: 0.185783, reader_cost: 0.00072 s, batch_cost: 0.10854 s, samples: 2560, ips: 2358.50732
[2020/12/21 16:54:00] root INFO: save model in ./output/rec/ic15/latest
[2020/12/21 16:54:00] root INFO: Initialize indexs of datasets:['./train_data/rec_gt_train.txt']
[2020/12/21 16:54:02] root INFO: epoch: [71/72], iter: 1210, lr: 0.000500, loss: 15.632069, acc: 0.000000, norm_edit_dis: 0.184799, reader_cost: 0.09936 s, batch_cost: 0.17991 s, samples: 1024, ips: 569.16633
[2020/12/21 16:54:03] root INFO: epoch: [71/72], iter: 1220, lr: 0.000500, loss: 15.758248, acc: 0.000000, norm_edit_dis: 0.184083, reader_cost: 0.00022 s, batch_cost: 0.08929 s, samples: 2560, ips: 2867.04557
[2020/12/21 16:54:04] root INFO: save model in ./output/rec/ic15/latest
[2020/12/21 16:54:04] root INFO: best metirc, acc: 0
```

**Comment by littletomatodonkey** [comment]
The training images scale is so small, maybe more data and more iterations are better.

**Comment by vinhtq115** [comment]
> The training images scale is so small, maybe more data and more iterations are better.

Okay I will try increasing the iteration.
